### ISSUE TITLE
spelling mistake

### ISSUE THREAD

**Issue by RocketFlashXD** [kickz1997/discord.js-Moderation-Bot#6]
Just want to say that I found a spelling mistake in "warn.js" on line 22.
It says "message.channe.send"
;)

**Comment by DeathHound6** [comment]
Hey @RocketFlashXD thanks for bringing this to our attention! 
It's been a long time since any of us have worked on this code, but I'll try to push a fix to this at some point

**Comment by KainesLaw** [comment]
Don't bother with it @DeathHound6. This code is written in v11 so there is no use in it.

**Comment by DeathHound6** [comment]
@ToonDiscord 
That is true, seeing as v11 is going to become deprecated when v13 is released, but the code could always be updated to v12 or v13 when it does release
